### ISSUE TITLE
LivePlot and relative scans are confusing.

### ISSUE THREAD

**Issue by ambarb** [bluesky/bluesky-widgets#18]
'ophyd': '1.5.1'
'bluesky': '1.6.3'

Workflow:
* Relative scan on nanop.tz (+/- 0.0025)  --> blue 133031
* Relative scan on nanop.tz (-0 , 0.005)   --> orange 133032
* Moved relative +0.0026
* Scanned a different direction for same assembly using different motor (nanop.tx) and moved slightly
* Relative scan on nanop.tz  (+/- 0.0025)  --> green 133034

From the attached plot, it is clear that what is plotted must be absolute values, and not relative values as indicated by the labels on the axes.  This will become very confusing.  I myself, would prefer absolute values becuase `bec.peaks` reports absolute values.  Relative is also nice in utility though (especially if you want to move to a local maximum, minimum or something not in `bec.peaks`.  

```python
RE(mv(fccd.fccd1.capture_bgnd, 1))
RE(rel_scan([fccd], nanop.tz,-0.0025, 0.0025, 101))
RE(rel_scan([fccd], nanop.tz,0.0, 0.0050, 101))
#RE(ZP_mvr(0.0026, dir='z'))
RE(ZP_mvr(0.0026, dir='z'))
grabit
RE(rel_scan([fccd], nanop.tx,-0.0025, 0.0025, 101))
RE(ZP_mvr(-0.0018, dir='x'))
grabit
RE(rel_scan([fccd], nanop.tz,-0.0025, 0.0025, 101))
```

**Comment by danielballan** [comment]
Would it be suitable for the best-effort visualization code to detect when an axis is begin traversed in a "relative" fashion by a plan and, if so, use a [secondary axis](https://matplotlib.org/3.1.0/gallery/subplots_axes_and_figures/secondary_axis.html)?

**Comment by ambarb** [comment]
> Would it be suitable for the best-effort visualization code to detect when an axis is begin traversed in a "relative" fashion by a plan and, if so, use a [secondary axis](https://matplotlib.org/3.1.0/gallery/subplots_axes_and_figures/secondary_axis.html)?

Yes, if I understand correctly.  One x-axis is absolute and another x-axis would have relative.  but I don't see how you do that in the same subplot in the circumstance I outlined above --  if you move after the first scan and then do another relative scan.  IT seems to me that this would have the same issue.

Maybe it is just better to accept the behavior as it stands because for the `real_grid_scan` and `rel_scan` it is also convenient to just `bps.mvr` to a desired position on the plot if it does not correspond to `bec.peaks` .  It is just super confusing to overlay plots that so the other option is to just make sure that `rel_scan` is plotted using relative coordinates and not absolute coordinates. Do I make sense?  The problem there would be that if there are "common" features to the scan, they will no longer match, but at least the axis makes sense.

**Comment by ambarb** [comment]
For the last case, if you want to plot absolute, open a notebook and plot absolute valves from the table.
